docs/installation/mailing.md

```diff
@@ -60,7 +60,7 @@ To set Autolab up to use a custom SMTP Server, edit the following in `production
         
       Refer to the SMTP settings instructions that your selected service provides you such as [SendGrid SMTP for Ruby on Rails](https://sendgrid.com/docs/for-developers/sending-email/rubyonrails/), [Amazon SES](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-smtp.html).
 
-3. Update the "from" setting
+3. Update the "from" setting in `production.rb`
```

Comment:
This is just a question - but would it be useful to pull out the smpt settings out from production.rb as well?

Reply:
I think it would be useful to have all SMTP config centralized at some point.